tests/trx_generator/trx_generator_tests.cpp

```diff
@@ -524,19 +523,17 @@ BOOST_AUTO_TEST_CASE(simple_http_client_async_test) {
    int callbackCalledCnt = 0;
 
    // Launch the asynchronous operation
-   // std::future<http::response<http::string_body>> future_response = std::make_shared<session>(ioc)->run(host,
-   // port_str, target, version, content_type, test_body);
    http_client_async::async_http_request(
-       params, test_body,
+       params, std::move(test_body),
        [test_body, &callbackCalledCnt](boost::beast::error_code ec, http::response<http::string_body> response) {
```

Comment:
Can't move `test_body` as you capture it in the lambda.

Reply:
Addressed: [53a720b](https://github.com/AntelopeIO/leap/pull/1138/commits/53a720bc157b9ef9ade67ea15b929b0d1e21c14a), [2bd0f23](https://github.com/AntelopeIO/leap/pull/1138/commits/2bd0f23cd52cd5f372c500b1e0c586ce5deb3c1c),[815ac6e](https://github.com/AntelopeIO/leap/pull/1138/commits/815ac6e55f168bf2205251f5f1b165f8024e479c)